sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -214,7 +214,7 @@ Listens for incoming BAPI functions.
 | Name | Type | Description | Default Value | Required
 | Configuration | String | Name of the configuration to use. | | x
 | Gateway Host a| String |  Name of the host running the gateway server. This is the name of the host running the gateway handler. SAP Gateway is a development framework used to connect non-SAP applications to SAP applications. You can configure the SAP Gateway host to be the trusted system using an RFC destination.|  | x
-| Gateway Service a| String |  Either the name or service port for the gateway service of the SAP system. |  | x
+| Gateway Service a| String |  Service port for the gateway service of the SAP system. To use name instead of port, additional mapping of gateway service name :: gateway service port is needed. |  | x
```

Comment:
```suggestion
| Gateway Service a| String |  Service port for the gateway service of the SAP system. To use the name instead of the port, you must map the gateway service name to the gateway service port.  |  | x

```